.github/workflows/release.yml

```diff
@@ -164,6 +167,9 @@ jobs:
             cpu: ARM64
           - arch: windows-64
             name: Windows-X64
+        exclude:
```

Comment:
This exclude is not needed – only 3 default configurations are created, and linux-arm64 is not in the list.